pages/stems/[id].tsx

```diff
@@ -158,9 +190,14 @@ export const getServerSideProps: GetServerSideProps = async context => {
 	const stemId = context.query.id
 	const res = await get(`/stems/${stemId}`)
 	const data: IStemDoc | null = res.success ? res.data : null
+
+	// Get all Projects
+	const result = await get(`/projects`)
+	const projects: IProjectDoc[] | null = result.success ? result.data : null
```

Comment:
Obviously this is best suited like recent project activity API handler, using mongoose helper methods similarly. Then the route can be `/api/stems/[id]/collaborations` which can return the array of projects so it's easily consumable by the client.  And then can just keep this JSX code as such and update the API route.